.github/update_lag.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Lag Status
+
+on: 
+  workflow_dispatch:
+    inputs:
+#      isRetrigger:
+#        description: 'Re-trigger a failed dependency bump workflow'
+#        required: false
+#        default: 'false'
```

Comment:
Is this necessary?

Reply:
8bc3836c253f9584162fce9532c843628a4c9cf6